pkg/quarterdeck/mock/quarterdeck.go

```diff
@@ -64,7 +64,7 @@ func (s *Server) routeRequest(w http.ResponseWriter, r *http.Request) {
 	case path == StatusEP:
 		s.handlers[path](w, r)
 	case path == RegisterEP:
-		s.handlers[path](w, r)
+		s.handlers[RegisterEP](w, r)
```

Comment:
```suggestion
		s.handlers[path](w, r)
```

Is this stray from the changes to the domain?